docs/reference/commandline/network_inspect.md

```diff
@@ -201,6 +201,97 @@ $ docker network inspect ingress
 ]
 ```
 
+### Using `verbose` option for `network inspect`
+
+`docker network inspect --verbose` for swarm mode overlay networks shows service-specific details like VIP and port mappings. It also
```

Comment:
Maybe "...like the service's VIP"? Sorry to keep nitpicking this. The sentence just doesn't sound correct without something between "like" and "VIP".